wts/blueprints/tokens.py

```diff
@@ -21,4 +22,9 @@ def get_token():
         except ValueError:
             return flask.jsonify({"error": "expires has to be an integer"}), 400
 
-    return flask.jsonify({"token": get_access_token(requested_idp, expires=expires)})
+    try:
+        access_token = get_access_token(requested_idp, expires=expires)
+    except Exception:
+        traceback.print_exc()
```

Comment:
maybe use the logger to print this? Curious though why this was necessary, if there was an exception it should've been raised and eventually (unless it's being caught and masked in a calling function, in which case maybe the caller needs to print the resulting error).

Reply:
I can dig into this more, but this is also a change from #40

https://github.com/uc-cdis/workspace-token-service/pull/36#discussion_r937016968